src/monitoring/monitorIndexer.js

```diff
@@ -0,0 +1,68 @@
+import {
+    getDataToMonitorStatus
+} from './getStatus.js';
+import { createRequire } from "module";
+import {SesClient} from "./emailClient.js"
+import Constant from './config/constants.js';
+
+
+const require = createRequire(import.meta.url);
+const fullConfig = require('./config/config.json');
+let allNetworkIndexer = fullConfig['network_index']
+
+
+export class MonitorIndexer {
+    static async monitoringIndexer() {
+        for (const [network, config] of Object.entries(allNetworkIndexer)) {
```

Comment:
the for loop is very long and not readable.
Can it be split into smaller logical methods?